src/components/SourcesTree.js

```diff
@@ -123,6 +129,11 @@ let SourcesTree = React.createClass({
     const { focusedItem, sourceTree, parentMap } = this.state;
     const isEmpty = sourceTree.contents.length === 0;
 
+    if (isEnabled("showSource")) {
+      const { shownSource } = this.props;
+      shownSource;
```

Comment:
not much, it shows where the source tee will get the data.

i believe @arthur801031 knows what to do with it!